shared/wallets/transaction/index.tsx

```diff
@@ -212,7 +212,7 @@ const Detail = (props: DetailProps) => {
       {' '}
       by converting{' '}
       <Text type={textTypeExtrabold}>
-        {props.sourceAmount} {props.sourceAsset}
+        {props.sourceAmount} {props.sourceAsset || 'XLM'}
```

Comment:
This gives us transaction messages like:
![Screen Shot 2019-06-26 at 11 25 09 AM](https://user-images.githubusercontent.com/5677971/60193141-24baef80-9805-11e9-94ef-76a09d25da2a.png)

I'm not sure if we'd also want to include a local currency conversion for XLM in lumen<->non-native asset conversions.

Reply:
we don't need the local currency conversion for XLM in path payments.